cognite_toolkit/_cdf_tk/client/api/locations/location_filters.py

```diff
@@ -0,0 +1,129 @@
+
+    @overload
+    def create(self, location_filter: LocationFilterWrite) -> LocationFilter: ...
+
+    @overload
+    def create(self, location_filter: Sequence[LocationFilterWrite]) -> LocationFilterList: ...
+
+    def create(
+        self, location_filter: LocationFilterWrite | Sequence[LocationFilterWrite]
+    ) -> LocationFilter | LocationFilterList:
+        """Create a new LocationFilter.
+
+        Args:
+            LocationFilter: LocationFilterWrite or list of LocationFilterWrite.
+
+        Returns:
+            LocationFilter or LocationFilterList
+
+        """
+
+        if isinstance(location_filter, LocationFilterWrite):
+            res = self._post(url_path=self._RESOURCE_PATH, json=location_filter.dump())
+            return LocationFilter._load(res.json(), cognite_client=self._cognite_client)
+
+        return LocationFilterList(
+            [
+                LocationFilter._load(
+                    self._post(url_path=self._RESOURCE_PATH, json=item.dump()).json(),
+                    cognite_client=self._cognite_client,
+                )
+                for item in location_filter
+            ]
+        )
+
+    def retrieve(self, external_id: str) -> LocationFilter | None:
+        for loc in self.list():
+            if loc.external_id == external_id:
+                return loc
+        raise CogniteNotFoundError(not_found=[external_id])
+
+    @overload
+    def update(self, location_filter: LocationFilterWrite) -> LocationFilter: ...
+
+    @overload
+    def update(self, location_filter: Sequence[LocationFilterWrite]) -> LocationFilterList: ...
+
+    def update(
+        self, location_filter: LocationFilterWrite | Sequence[LocationFilterWrite]
+    ) -> LocationFilter | LocationFilterList:
+        if isinstance(location_filter, LocationFilterWrite):
+            res = self._put(
+                url_path=f"{self._RESOURCE_PATH}/{self._ids[location_filter.external_id]}",
+                json=location_filter.dump(),
+            )
+            return LocationFilter._load(res.json(), cognite_client=self._cognite_client)
+
+        return LocationFilterList(
+            [
+                LocationFilter._load(
+                    self._put(url_path=f"{self._RESOURCE_PATH}/{self._ids[item.external_id]}", json=item.dump()).json(),
+                    cognite_client=self._cognite_client,
+                )
+                for item in location_filter
+            ]
+        )
+
+    def delete(self, ids: int | Sequence[int]) -> int:
+        if isinstance(ids, int):
+            ids = [ids]
+
+        n = 0
+        for id in ids:
+            self._delete(url_path=f"{self._RESOURCE_PATH}/{id}")
+            n += 1
+        return n
+
+    def list(self) -> LocationFilterList:
+        res = self._post(url_path=self._RESOURCE_PATH + "/list", json={"flat": False})
+        return LocationFilterList._load(res.json()["items"], cognite_client=self._cognite_client)
```

Comment:
There is a method `self._list(...)` that typically we use. It is not necessary as there is no paging for this endpoint, but I would requests that from the backend team